### ISSUE TITLE
Default values when using DSLVariables.

### ISSUE THREAD

**Issue by luisvicentesanchez** [graphql-python/gql#302]
As an user, I sometimes write queries this way:

```
gql("""query SomeQuery($arg: Int = 100) {
  field(input=$arg) {
    ...
  }
}""")
```
That gives me some flexibility as I can execute this query without passing any variables. I haven't found how to do this using the DSL in the documentation. Is it fair to assume that this is not supported? It not, would it make sense to add support for it?

**Comment by leszekhanusz** [comment]
This is not supported for now and you're right, we should support it.

**Comment by leszekhanusz** [comment]
Please check PR #322

**Comment by leszekhanusz** [comment]
Fixed in [v3.3.0](https://github.com/graphql-python/gql/releases/tag/v3.3.0)
